delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/organizations/Organization.scala

```diff
@@ -35,4 +49,7 @@ object Organization {
   val context: ContextValue                                           = ContextValue(contexts.organizations)
   implicit val organizationJsonLdEncoder: JsonLdEncoder[Organization] =
     JsonLdEncoder.fromCirce(context)
+
+  implicit val metadataEncoder: Encoder.AsObject[Metadata]    = deriveConfiguredEncoder[Metadata]
```

Comment:
I think this one should be private and only the `JsonLdEncoder[Metadata]` public